cmake/modules/hunter_windows_utils.cmake

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov
+# All rights reserved.
+
+# CMAKE function to convert windows log filename to 8.3 filename
+function(hunter_windows_to_snf_filename path new_path_variable)
+  set(conversion_cmd
+    "cmd" 
+    "/c" 
+    "for" 
+    "%A" 
+    "in" 
+    ("${path}") 
+    "do" 
+    "@echo"
+    "%~sA"
+	)
+  
+  execute_process(
+    COMMAND ${conversion_cmd}
+    RESULT_VARIABLE conversion_result
+    OUTPUT_VARIABLE converted_path
+  )
+  
+  if(NOT conversion_result EQUAL 0)
+    hunter_fatal_error("Command to convert path " ${path} " to 8.3 Filename failed. Command is " ${conversion_cmd})
+  endif()
+  
+  string(REGEX REPLACE "\n" "" converted_path ${converted_path})
+  string(REGEX REPLACE "[\\]" "/" converted_path ${converted_path})  
+
+  set(${new_path_variable} "${converted_path}" PARENT_SCOPE)
+endfunction()
+
+#Set command to change environment variables to 8.3 filenames to windows
+function(hunter_windows_env_command variable_name)
```

Comment:
Please use separate file for this function. See description of idea: https://github.com/ruslo/0/wiki/CMake#library-of-cmake-extra-modules